### ISSUE TITLE
low contrast text on saving 

### ISSUE THREAD

**Issue by CleezyITP** [toolness/pode#18]
The text that reads "Your code has been saved." has a low contrast ratio with the background of the editor and does not meet WCAG standards. 

<img width="583" alt="screen shot 2017-11-09 at 9 44 27 am" src="https://user-images.githubusercontent.com/5375410/32611106-b6fd42fa-c532-11e7-8821-6a70e1db283f.png">
